### PR TITLE
Edits to BigAnimal PR5376

### PR DESCRIPTION
**Pull request by ebgitelman**

### CODE REVIEW

File: product_docs/docs/biganimal/release/getting_started/identity_provider/index.mdx [hunk 1]
```diff
@@ -78,23 +78,22 @@ Once your identity provider is set up, you can view your connection status, ID,
...
-   - Add a TXT record.
-   - In the Name field, enter @.
-   - In the Value field, enter the verification string provided, eg. 
-   - “edb-biganimal-verification=VjpcxtIC57DujkKMtECSwo67FyfCExku”
-   - Save your changes and wait for the DNS propagation period to complete. This can take up to 48 hours.
+   1. Log in to your domain registrar or web host account.
+   1. Navigate to the DNS settings for the domain you want to verify.
+   1. Add a TXT record.
+   1. In the **Name** field, enter `@`.
+   1. In the **Value** field, enter the verification string provided, for example, `"edb-biganimal-verification=VjpcxtIC57DujkKMtECSwo67FyfCExku"`.
```
Comment (ebgitelman) [inline product_docs/docs/biganimal/release/getting_started/identity_provider/index.mdx:87]: Is this supposed to have the quotes? It had quotes in the original, and I wanted to just tag it as code, but I wanted to verify the string does include the quote marks. @edcrewe

Comment (ebgitelman) [inline product_docs/docs/biganimal/release/getting_started/identity_provider/index.mdx:87]:
```suggestion
   1. In the **Value** field, enter the verification string provided, for example, `edb-biganimal-verification=VjpcxtIC57DujkKMtECSwo67FyfCExku`.
```

**Comment by ebgitelman** [comment]
@nidhibhammar This needs your approval. Thanks!

**Review by nidhibhammar — APPROVED** [review]
LGTM

**Pull request merged**